About.html

```diff
@@ -4,10 +4,136 @@
+        <p>Sponsor</p>
+        <p>News</p>
+        <p><button type="button" class="menu-btn">Coupe du Monde</button></p>
+      </div> 
+      
+      <!--Hamburger Menu--> 
+      <div class="menu__toggler desktop-display"> 
+        <span></span>
+      </div>
+    </header>  
+    <!-- Main Container -->
+    <main class="main-container">
+        <!-- Information -->
+        <section class="about-info">
+            <div class="info-container">
+            <h2 class="cheers">Hello Climbers!</h2>
+            <h1 class="heading">Coupe Du Monde <br> in Chamonix <span>2022</span></h1>
+            <!-- <div class="underlinefull"></div> -->
+            <div class="infoWrapper">
+                <div class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium voluptate saepe officia quam, ad neque odit nobis! Natus, et quas dolorum quia tempore sunt totam aperiam possimus in eos a?
+                Culpa voluptates illum harum modi voluptas necessitatibus eligendi </div>
+             </div>
+             <div class="info-contact">
+                <p>Please contact us per email for any further questions
+                    about Summer Coupe du Monde 2022!</p>
+                <p class="info-email"><strong>@ifsc-climbing.org</strong></p>    
+             </div>
+            </div>
+        </section>
+        <!-- Logo with Banner-->
+        <section class="about-logo">
+            <h3>Coupe Du Monde <br> Chamonix <span>2022</span> Logo</h3>
+            <div class="underlinefull"></div>
+            <p class = "for-mobile">Lorem ipsum dolo it amet consectetur adipisicing elit. Veniam a hic aliquid </p>
+            <p class="for-desktop">Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam a hic aliquid fugiat corrupti dignissimos aspernatur amet atque distinctio consequuntur, culpa magni natus quaerat nesciunt. Nemo totam perferendis saepe doloremque!
+            Sequi corporis rerum eaque voluptate commodi obcaecati, soluta similique debitis id veniam, magni, ducimus rem odio. Eos, hic tempora adipisci minus provident accusantium veniam ratione possimus,  </p>
+           <div class="aboutlogoimg">
+               <img src="media/chamonix.png" alt="Website Logo">
+           </div>
+           <div class="border"></div>
+        </section>
+        <!-- Past Events -->
+        <section class="about-past">
+            <h3>See the Past Coupe Du Monde</h3>
+            <div class="underlinefull"></div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.  delectus!
+            </p>
+            <div class="Container">
+                <img src="media/pasteventtwo.jpg" class= "image1" alt="Past event1">
+                <div class="overlay1">Climbing World Cup 2020</div>
+                <div></div>
+                <img src="media/pasteventtwo.jpg"  class= "image2" alt="Past event2">
+                <div class="overlay2">Climbing World Cup 2018</div>
+            </div>
```

Comment:
[ OPTIONAL ] Images are blur here and it's difficult to see overlay text on them. Kindly make the text and images clear.
![images](https://user-images.githubusercontent.com/35504149/137342908-e275d0e6-6bf5-4bc8-aacd-284183301ee1.png)